docs/networkx-integration.md

```diff
@@ -12,17 +12,81 @@ Below there are few useful examples to follow.
 
 ## Parse graph, show it
 
-To load graph form a `.jelly` file, and (optionally) calculate some practical graph characteristics, see:
+Let's investigate worldwide political connections (support and oposition relations)!
+We are given a graph in `.jelly` containing information about political stances extracted from news articles.  
+Let's dive in and get some useful information!
 
-{{ code_example('networkx_integration/01_parse_calculate_visualize.py') }}
+We can easily load it:  
 
-which converts an RDFLib graph to NetworkX, computes key metrics and visualizes it.
+{{ snippet_admonition('examples/networkx_integration/01_parse_calculate_visualize.py', 24, 27, title="Loading") }}
+
+???+ tip "Output from print()"
+    ```text
+    Loaded graph with 90000 instances.
+    ```
+
+Convert it into a convenient NetworkX graph:
+
+{{ snippet_admonition('examples/networkx_integration/01_parse_calculate_visualize.py', 55, 56, title="Converting") }}
+
+Is our graph fully connected? It's important to know (are all political relations tied togehter?), let's check here:
+
+{{ snippet_admonition('examples/networkx_integration/01_parse_calculate_visualize.py', 58, 60, title="Fully connected check") }}
+
+???+ tip "Output from print()"
+    ```text
+    Connected components: 24
+    ```
+
+Which nodes are connected the most (have most connections?), let's see top 5 of them:
+
+{{ snippet_admonition('examples/networkx_integration/01_parse_calculate_visualize.py', 62, 66, title="Top 5 nodes by degree") }}
+
+???+ tip "Output from print()"
+    ```text
+    Top 5 nodes sorted by degree:
+    Socrates: 241
+    Cavaco: 189
+    Passos Coelho: 187
+    Costa: 179
+    Antonio Costa: 168
+    ```
+
+What is the shortest path between two nodes? We can check:
+
+{{ snippet_admonition('examples/networkx_integration/01_parse_calculate_visualize.py', 75, 79, title="Shortest path") }}
+
+???+ tip "Output from print()"
+    ```text
+    Shortest path from Socrates to Obama: Socrates -> Marcelo Rebelo de Sousa -> Durao Barroso -> Obama
+    ```
+
+However afterall, its best to the full picture (for our example we truncate to 10 nodes for clarity):
```

Comment:
This line contains 4 typos...

"However,"

"after all"

"it's"

"to get the"

please do install a spellchecker or something like that...